platform-core-module/src/main/java/com/softicar/platform/core/module/start/page/AGStartPageMessage.java

```diff
@@ -0,0 +1,25 @@
+package com.softicar.platform.core.module.start.page;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.emf.object.IEmfObject;
+import java.util.List;
+
+public class AGStartPageMessage extends AGStartPageMessageGenerated implements IEmfObject<AGStartPageMessage> {
+
+	private static final int DEFAULT_MESSAGE_AMOUNT_TO_LOAD = 5;
```

Comment:
IMO, the `DEFAULT_MESSAGE_AMOUNT_TO_LOAD` is rather a UI topic, and `getLatestMessages` should rather get that number as parameter.